debian/mariadb-server-10.6.postinst

```diff
@@ -238,9 +238,9 @@ EOF
   ;;
 
   triggered)
-    if [ -x "$(command -v systemctl)" ]; then
-      systemctl daemon-reload
-    else
+    if [ -d /run/systemd/system ]; then
```

Comment:
should this additionally include the previous criteria like:
```
if [ -d /run/systemd/system ] && [ -x "$(command -v systemctl)" ]; then
```
?

Reply:
I can't see any scenario where this would be necessary but it's always a good idea to check that a command is available (and in the PATH). :+1:

Reply:
I don't think there's a need. `/run` is " volatile runtime data that does not persist across reboots " [1]. If `/run/systemd/system` exists, it means that the systemd system daemon is running, and it's like 99.99% sure that `systemctl` is present. I can't think of any scenario where that wouldn't be the case.

Adding the second test won't hurt either, but it would be redundant, IMO.

[1] https://debian-handbook.info/browse/stable/sect.filesystem-hierarchy.html

Reply:
Ok. Fine as is. It was considered so that's ok with me.

Reply:
@elboulangero I really like your usage of literature to confirm the facts of your statement and to explain to the audience. 👍

Reply:
Thanks for the kind words @an3l :)